lib/diplomat/kv.rb

```diff
@@ -256,7 +256,7 @@ def decode_transaction(transaction) # rubocop:disable Metrics/MethodLength
           next unless resp['KV']['Value']
           begin
             resp['KV']['Value'] = Base64.decode64(resp['KV']['Value'])
-          rescue # rubocop:disable RescueWithoutErrorClass
+          rescue StandardError
             nil
```

Comment:
May need to add a logging message

Reply:
FAR if someone wants to do it, I don't change the behaviour here, just fix rubicop since JSON parsing and base64 are StandardError